### ISSUE TITLE
Window menu shortcuts do not work when Japanese IME is active

### ISSUE THREAD

**Issue by kyamamoto03** [Microsoft/vscode#5515]
- VSCode Version:1.0.0
- OS Version:Windows7

ファイルを保存するときにAlt+F+Sを使いますが、日本語入力の時にSが反応しません。

**Comment by Tyriar** [comment]
I believe this issue is about not being able to access the menu options while the Japanese IME is active. <kbd>alt</kbd>+<kbd>f</kbd> (file menu) then <kbd>s</kbd> to save, but the IME has likely taken the s captive until it's completed a character.

**Comment by bpasero** [comment]
@kyamamoto03 can you please update the description in english?

**Comment by kyamamoto03** [comment]
When a file is saved , and are using the JapaneseIME Alt + F (FileMenu) + S (SaveAs) does not work

**Comment by bill-mybiz** [comment]
Are y'all working on this right now? I have VS code insiders, and the Alt + F won't bring up the menu at all (I'm not in any kind of localized scenario). VS code insiders 1.1.0-insider.

**Comment by Tyriar** [comment]
@bill-mybiz this issue is specifically about when an IME is active. If you're not using one you should create a new issue and provide OS and other relevant details about your configuration that could affect it.

**Comment by bpasero** [comment]
@kyamamoto03 does it still reproduce? can you try our insider builds? 
You can give our preview releases a try from: http://code.visualstudio.com/Download#insiders
